integration-cli/docker_cli_daemon_test.go

```diff
@@ -1448,7 +1449,7 @@ func (s *DockerDaemonSuite) TestCleanupMountsAfterDaemonAndContainerKill(c *chec
 	c.Assert(strings.Contains(string(mountOut), id), check.Equals, true, comment)
 
 	// kill the container
-	icmd.RunCommand(ctrBinary, "--address", "unix:///var/run/docker/libcontainerd/docker-containerd.sock", "containers", "kill", id).Assert(c, icmd.Success)
+	icmd.RunCommand(ctrBinary, "--address", "/var/run/docker/containerd/docker-containerd.sock", "--namespace", "docker", "tasks", "kill", id).Assert(c, icmd.Success)
```

Comment:
namespace: `docker` -> `moby`? `moby-core`?

Reply:
@mlaventure maybe put `"docker"` in a variable so we can change it easily.

Reply:
Will do, will call the namespace moby too while at it.